src/Codeception/Module/Yii2.php

```diff
@@ -122,67 +127,124 @@
  * Maintainer: **samdark**
  * Stability: **stable**
  *
+// * @property \Codeception\Lib\Connector\Yii2 $client
```

Comment:
?

Reply:
Commented that out to have my IDE track down the original definition.. shouldn't have been committed